### PR TITLE
Use same screenshot comparing mechanism as in cockpit

### PR DESCRIPTION
**Pull request by alexsuter**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -61,77 +60,19 @@ pipeline {
...
-              manualDeploy = input(
-                message: 'Screenshot comparison failed. Please check them manually.', parameters: [
-                [$class: 'BooleanParameterDefinition', defaultValue: false, description: '', name: 'Deploy screenshots?']])
-            }
-          }
-        }
-      }
-    }
-
-    stage('deploy maven') {
```
Comment (alexsuter) [inline Jenkinsfile:119]: do not split uncessery maven executions. maven is not designed for that.

File: Jenkinsfile [hunk 2]
```diff
@@ -163,3 +104,10 @@ pipeline {
 def isReleaseOrMasterBranch() {
   return env.BRANCH_NAME == 'master' || env.BRANCH_NAME.startsWith('release/')
 }
+
+def getScreenshotRefBranch() {
```
Comment (alexsuter) [inline Jenkinsfile:108]: same es in engine cockpit

File: maven/image-validation/pom.xml [hunk 3]
```diff
@@ -1,4 +1,6 @@
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (alexsuter) [inline maven/image-validation/pom.xml:1]: same as in engine cockpit

File: Jenkinsfile [hunk 4]
```diff
@@ -61,77 +60,19 @@ pipeline {
...
-        not {
-          triggeredBy 'TimerTrigger'
-        }
-      }
-      steps {
-        script{
-          if (currentBuild.currentResult == 'FAILURE') {
-            timeout(time: 10, unit: 'MINUTES')
-            {
-              manualDeploy = input(
```
[the remark this answers is not in the document]
Comment (ivy-lli) [reply Jenkinsfile:110]: jep possible. This also blocked a build node, so was an idea but wasn't worth I think...

**Review by ivy-lli — APPROVED** [review]
Looks good. maybe splitting into two pipelines would be nice too. But good first step I think

**Pull request merged**